src/main/java/org/openbase/bco/bcozy/view/UnitMenu.java

```diff
@@ -76,15 +104,21 @@ public UnitMenu(final double height, final double width) {
         roomInfo.getStyleClass().clear();
         roomInfo.getStyleClass().add("headline");
 
+        energyInfo = new Label("Select your Energy");
```

Comment:
Please use ObserverLabel instead and declare it at the multi lang bundle.

Reply:
Removed during some of the last commits